docs/modules/hdfs/pages/usage-guide/operations/pod_disruptions.adoc

```diff
@@ -0,0 +1,108 @@
+
+= Pod disruptions
+
+You can configure Pod disruptions for HDFS nodes as described in xref:concepts:operations/pod_disruptions.adoc[].
```

Comment:
Same here

Reply:
Same above xD

Reply:
...."configure Pod disruptions" just sounds bad to me

Reply:
Yeah, agreed.
What about `You can influence Pod disruptions` or `You can control Pod disruptions` ?

Reply:
"To prevent pod disruptions, you can configure PodDisruptionBudgets" maybe?

Reply:
I agree that this (currently) far easier for the user to understand. But I see problems with this in the future as described in my comment

> I would consider this a implementation detail. In the future we might want something more smart than PDBs, e.g. k8s asking the hdfs-operator "can i safely evict this pod?".
This would still count as Pod disruptions, but not PDB.

Reply:
Then we can change the docs in the future.
Configure Pod disruptions just reads weird to me. But I'm fine with keeping it.